repl/src/test/scala/ammonite/repl/AdvancedTests.scala

```diff
@@ -260,6 +260,17 @@ object AdvancedTests extends TestSuite{
         error: not found: value x
       """)
     }
+    'compilerPlugin{
+      check.session("""
+        @ load.plugin.ivy("org.spire-math" %% "kind-projector" % "0.6.3")
```

Comment:
Can we add a negative test to ensure that `load.ivy`ing `kind-projector` without the `plugin` bit will fail to activate the plugin, and that `TC[Either[String, ?]]` results in a compile error? Could even be part of the same `check.session`